Cargo.toml

```diff
@@ -15,13 +15,13 @@ lto = true
 codegen-units = 1
 
 [features]
-i3 = ["i3ipc"]
+i3 = ["swayipc"]
 
 [dependencies]
 cairo-sys-rs = "0.14"
 css-color-parser = "0.1"
 font-loader = "0.11"
-i3ipc = { version = "0.10", optional = true }
+swayipc = { git = "https://github.com/JayceFayne/swayipc-rs", optional = true }
```

Comment:
Can't we use a proper release here?

Reply:
Resolved with explicit dependency on 3.0.0